addons/capetrader/capetrader.lua

```diff
@@ -0,0 +1,495 @@
+        windower.add_to_chat(123, 'You have not yet setup your cape and augment information with the //ct prep command!')
+    else
+        currentlyAugmenting = false
+    end
+end
+
+function checkAugLimits()
+    local capeItem
+    for index, item in pairs(inventory) do
+        if index ~= 'max' and index ~= 'count' and index ~= 'enabled' then
+            if item.id == currentCape.id then
+                capeItem = item
+                break
+            end
+        end
+    end
+
+    local augmentTable
+    if extData.decode(capeItem).augments then
+        augmentTable = extData.decode(capeItem).augments
+    end
+
+    local augValue
+    if augmentTable then
+        if string.contains(string.lower(pathItem.en), 'thread') then
+            augValue = augmentTable[threadIndex]
+        elseif string.contains(string.lower(pathItem.en),'dust') then
+            augValue = augmentTable[dustIndex]
+        elseif string.contains(string.lower(pathItem.en), 'dye') then
+            augValue = augmentTable[dyeIndex]
+        elseif string.contains(string.lower(pathItem.en), 'sap') then
+            augValue = augmentTable[sapIndex]
+        end
+    else
+        augValue = 'none'
+    end
+
+    if string.lower(augValue) == 'none' or not augmentTable then
+        return 'empty'
+    end
+
+    local max = maxAugMap[maxAugKey].max
+    if string.contains(augValue, max) then
+        windower.add_to_chat(123, 'You have augmented your ' .. currentCape.en .. ' to the max already with ' .. pathItem.en .. '.')
+        return 'maxed'
+    end
+
+    local mustContainTable = maxAugMap[maxAugKey].mustcontain
+    for k, augmentString in pairs(mustContainTable) do
+        if not string.contains(string.lower(augValue), string.lower(augmentString)) then
+            windower.add_to_chat(123,'You can\'t augment your ' .. currentCape.en .. ' with ' .. pathName .. ' because it has already been augmented with: ' .. augValue .. ' using ' .. pathItem.en .. '.')
+            return 'notmatching'
+        end
+    end
+
+    local cantContainTable = maxAugMap[maxAugKey].cantcontain
+    if table.length(cantContainTable) > 0 then
+        for k, augmentString in pairs(cantContainTable) do
+            if string.contains(string.lower(augValue), string.lower(augmentString)) then
+                windower.add_to_chat(123,'You can\'t augment your ' .. currentCape.en .. 'with ' .. pathName .. ' because it has already been augmented with: ' .. augValue .. ' using ' .. pathItem.en .. '.')
+                return 'notmatching'
+            end
+        end
+    end
+
+    return 'allclear'
+end
+
+function injectAugmentConfirmationPackets()
+    local optionIndex
+    if firstTimeAug then
+        firstTimeAug = false
+        optionIndex = 512
+    else
+        optionIndex = 256
+    end
+
+    local packet = packets.new('outgoing', 0x05B)
+    packet["Target"] = gorpaID
+    packet["Option Index"] = optionIndex
+    packet["_unknown1"] = pathIndex
```

Comment:
I've made a couple of small addons similar to this one that use this 0x05B packet as well. It seems like "_unknown1" will take values other than zero if the 0x05B dialog packet you are sending results in you exiting the related menu in game. Although it seems there are a few exceptions to this in the capetrader addon, for example unknown1 will be zero if you augment a cape and exit the menu with any of the following pairs of items/augmentpaths: thread/hp, dust/acc+atk, sap/wsd, dye/hp. 

I didn't add a note to the packets/fields.lua file because I'm not entirely sure how unknown1 works. I think if there are multiple dialog options that can result in exiting a menu unknown1 won't necessarily be equal to zero and can take different values. If you think that is plausible and worth adding to the fields file let me know and I can add an explanatory comment of some kind.